src/pineko/theory.py

```diff
@@ -22,6 +22,32 @@
 logger = logging.getLogger(__name__)
 
 
+def read_grids_from_nnpdf(dataset_name):
```

Comment:
Maybe this function can be moved outside theory. I would create an `utils.py` file and put it there

Reply:
agreed (in case you foresee a growth there also `nnpdf_.py` could be a good name)

Reply:
I haven't done it here because it would be a mess with more stuff, but actually this should be `read_grids` and used everywhere, and the function internally should choose (according to the config) where to read from and how to create the grids.

At the moment fonll and theory do it in a different way (and kfactor in yet another way) so that would be a bigger change, but something to keep in mind.